ams/server/src/main/java/com/netease/arctic/server/utils/CompressUtil.java

```diff
@@ -58,6 +64,9 @@ public static byte[] unGzip(byte[] bytes) {
       while ((len = gzipInputStream.read(buffer)) > 0) {
         byteArrayOutputStream.write(buffer, 0, len);
       }
+    } catch (ZipException e) {
+      LOG.warn("Fail to unzip, return original bytes", e);
+      return bytes;
```

Comment:
I guess this was done to be compatible with data that's not compressed with Gzip. I think such compatibility would be more appropriate to be placed in the Object2ByteArrayConvert.

Reply:
This method is used by Object2ByteArrayConvert and TaskFilesPersistence#loadTaskInputs. Should I add try-catch to both of them?

Reply:
yeah